src/sql/workbench/contrib/editData/browser/media/editData.css

```diff
@@ -7,6 +7,12 @@
 	box-sizing: border-box;
 }
 
+.sash-visible .monaco-sash {
+	border-bottom-color:rgb(81,81,81);
```

Comment:
Did you test this in both light/dark mode to make sure it meets accessibility contrast requirements?

Why are we using a custom sash here anyways instead of the VS provided one?

Reply:
The custom sash class used in editdataeditor allows for it to be dragged around in the editor , whereas the default sash class does not have such capabilities built in.

Reply:
What you mean "dragged around the editor"? The VS Code sashes can be moved around

![Capture](https://user-images.githubusercontent.com/28519865/78188317-dfc38980-7424-11ea-8cb9-32f53dc7517e.gif)

Reply:
The appearance for the sash bar is actually the border for the panel itself rather than that of the sash, so the css needs to be applied to the border of the container rather than anything else.

Reply:
@Charles-Gagnon From the top of the custom sash class description, it says that it allows classes using the Sash to have "UI logic that is agnostic to the orientation of the sash"